example/rlottiePlayer/rlottiePlayer.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "resource.h"
+#include <Commdlg.h>                        // OPENFILENAME
```

Comment:
Very bad coding convention....
See how comments indentation size is different.
You can keep the code consistency.
from)
A______//xxx
B__//eee
to)
A___//xxx
B___//xxx